files/initramfs-tools/resize-rootfs

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+case $1 in
+    prereqs)
+        exit 0
+        ;;
+esac
+
+# Extract kernel parameters
+set -- $(cat /proc/cmdline)
+for x in "$@"; do
+    case "$x" in
+        root=*)
+            root_dev="${x#root=}"
+            ;;
+        resize-rootfs)
+            [ -z "$root_dev" ] && exit 0
+            resize2fs -f $root_dev
```

Comment:
resize-rootfs may come after root label. so, can you move resize2fs out of the for loop?